[2022-07-13 11:32:28,529] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Demo.Container_creation manual__2022-07-13T11:32:24.780623+00:00 [queued]>
[2022-07-13 11:32:28,560] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Demo.Container_creation manual__2022-07-13T11:32:24.780623+00:00 [queued]>
[2022-07-13 11:32:28,561] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-13 11:32:28,562] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-13 11:32:28,563] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-13 11:32:28,769] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): Container_creation> on 2022-07-13 11:32:24.780623+00:00
[2022-07-13 11:32:28,782] {standard_task_runner.py:52} INFO - Started process 1225 to run task
[2022-07-13 11:32:28,792] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Demo', 'Container_creation', 'manual__2022-07-13T11:32:24.780623+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/demo.py', '--cfg-path', '/tmp/tmpm4vizlyr', '--error-file', '/tmp/tmpde9bqyky']
[2022-07-13 11:32:28,795] {standard_task_runner.py:80} INFO - Job 31: Subtask Container_creation
[2022-07-13 11:32:29,024] {task_command.py:370} INFO - Running <TaskInstance: Demo.Container_creation manual__2022-07-13T11:32:24.780623+00:00 [running]> on host 939e047cba2c
[2022-07-13 11:32:29,453] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Demo
AIRFLOW_CTX_TASK_ID=Container_creation
AIRFLOW_CTX_EXECUTION_DATE=2022-07-13T11:32:24.780623+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-13T11:32:24.780623+00:00
[2022-07-13 11:32:29,485] {base.py:68} INFO - Using connection ID 'demo' for task execution.
[2022-07-13 11:32:29,493] {logging_mixin.py:115} INFO - <***.providers.microsoft.azure.hooks.wasb.WasbHook object at 0x7f556b934590>
[2022-07-13 11:32:29,494] {logging_mixin.py:115} INFO - connection made successfully
[2022-07-13 11:32:29,502] {_universal.py:476} INFO - Request URL: 'http://azurite:10000/devstoreaccount1/demo?restype=REDACTED'
Request method: 'PUT'
Request headers:
    'x-ms-version': 'REDACTED'
    'Accept': 'application/xml'
    'User-Agent': 'azsdk-python-storage-blob/12.8.1 Python/3.7.13 (Linux-5.10.16.3-microsoft-standard-WSL2-x86_64-with-debian-11.3)'
    'x-ms-date': 'REDACTED'
    'x-ms-client-request-id': '7a675876-029f-11ed-8b94-0242ac120006'
    'Authorization': 'REDACTED'
No body was attached to the request
[2022-07-13 11:32:29,606] {_universal.py:504} INFO - Response status: 409
Response headers:
    'Server': 'Azurite-Blob/3.18.0'
    'x-ms-error-code': 'REDACTED'
    'x-ms-request-id': '536a51fb-83c6-4be4-a782-ef51aa317e3a'
    'content-type': 'application/xml'
    'Date': 'Wed, 13 Jul 2022 11:32:29 GMT'
    'Connection': 'keep-alive'
    'Keep-Alive': 'REDACTED'
    'Transfer-Encoding': 'chunked'
[2022-07-13 11:32:29,743] {wasb.py:371} INFO - Attempted to create container 'demo' but it already exists. If it is expected that this container will always exist, consider setting create_container to False.
[2022-07-13 11:32:29,744] {logging_mixin.py:115} INFO - container created successfully
[2022-07-13 11:32:29,745] {python.py:173} INFO - Done. Returned value was: 0
[2022-07-13 11:32:29,952] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Demo, task_id=Container_creation, execution_date=20220713T113224, start_date=20220713T113228, end_date=20220713T113229
[2022-07-13 11:32:30,132] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-13 11:32:30,364] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
